rustls/Cargo.toml

```diff
@@ -31,21 +90,8 @@ webpki = { workspace = true }
 pki-types = { workspace = true }
 zeroize = { workspace = true }
 zlib-rs = { workspace = true, optional = true }
-
-[features]
-default = ["aws_lc_rs", "logging", "std", "tls12"]
-
-aws-lc-rs = ["aws_lc_rs"] # Alias because Cargo features commonly use `-`
-aws_lc_rs = ["dep:aws-lc-rs", "webpki/aws_lc_rs"]
-brotli = ["dep:brotli", "dep:brotli-decompressor", "std"]
-custom-provider = []
-fips = ["aws_lc_rs", "aws-lc-rs?/fips"]
-logging = ["log"]
-read_buf = ["rustversion", "std"]
-ring = ["dep:ring", "webpki/ring"]
-std = ["webpki/std", "pki-types/std", "once_cell/std"]
-tls12 = []
-zlib = ["dep:zlib-rs"]
+# doc utility
+document-features = "0.2.10"
```

Comment:
Unfortunately it looks like `document-features` needs to be in `[dependencies]` for this to work. I would like to find another solution but don't know how long it would take. I will totally understand if we need to leave this part out.

Reply:
I'm afraid I don't think moving text from one file to another is a good justification for new dependencies. In general I would like to avoid proc-macro dependencies in rustls at all costs.